### DISCUSSION TITLE
Submissions

### DISCUSSION THREAD

**Discussion by ThomasFaria** [InseeFrLab/ESA-Nowcasting-2023#52] [General]
### **Entry 1**
**_RegARIMA model:_** automatic ARIMA modelling with detection of outliers and external regressors (financial series, macroeconomic series, toll traffic, google trends...). We use the RJdemetra package that provide an easy way to identify and estimate such models with high flexibility on parameters. RegARIMA models offer some advantages in this exercise :
    • they are relatively simple, easy to interpret, and so provide a useful benchmark for more complex models. 
    • we allow for automatic detection of outliers on the estimation period in order to avoid large bias on coefficients. Without adding outliers, "covid" points would for instance totally distort coefficients. Outliers identified through this procedure could in addition be used in other methods 
    • computation time is really fast (a few seconds for the whole set of countries) 
    • external regressors can be selected independantly through standard variable selection. 

### **Entry 2**
**_Dynamic Factor Model:_** using several sources of data (financial series, macroeconomic series, surveys, toll traffic, google trends…) we estimate a dynamic factor model per country. Missing data are filled, collinearity is taken into account, data are scaled and log transformed when necessary. In order to reduce the estimation burden we restrict the number of factor to a maximum of 4. Estimation takes generally less than 2 seconds per country. Exact timing can be found in the actions of our open-source repository (42 sec for all countries).

### **Entry 3**
**_Exponential smoothing models:_** forecasts are linear combinations of past values, with the weights decaying exponentially as the observations get older. Estimation takes 1 second by country. No data processing.

### **Entry 4**
**_XGboost model:_** using several sources of data (financial series, macroeconomic series, surveys, toll traffic, google trends…) we construct a dataset in which each row corresponds to a value per country and per date with many explicative variables. On this basis, we train a XGboost model to predict the value of the following month.

### **Entry 5**
**_LSTM model:_** using several sources of data (financial series, macroeconomic series, surveys, toll traffic, google trends…) we construct a dataset in which each row corresponds to a value per country and per date with many explicative variables. On this basis, we train a LSTM model to predict the value of the following month, hoping that it will learn from the time series.

**Comment by antoine-palazz** [comment]
Add a few words about reproductibility
